src/Game.tsx

```diff
@@ -151,6 +148,11 @@ function GameReview(props: {
       <div className="final-score-container">
         <h2 className="final-score">Score: {score}</h2>
       </div>
+      <h2 className="new-high-score">
+        {highScores.length === 0 || score >= highScores[0]
+          ? "New high score!"
+          : `Best score: ${highScores[0]}`}
```

Comment:
Can we remove the `Best score` when no new high score is set? E.g. we display a message on a new high score but nothing else. Just a UX personal preference

Reply:
That makes so much sense. Thanks for the comment.